modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -6,53 +6,236 @@
+- `scopes` - depending on the setup and needs of the Identity Provider, the list of required scopes is entered here
+- `provider-params` - additional configuration depending on the Identity Provider can be entered here (usually only necessary if the Identity Provider does not support service discovery)
+- `auth-params` - additional parameters which are sent to the Identity Provider during the auth requests
+- `use-token-introspection-endpoint` - if set to `true`, the token introspection endpoint is used to verify a given access token (only needed if the access token is not a JWT)
+- `token-introspection-endpoint-client-id` - the client id to be used with the token introspection endpoint
+- `token-introspection-endpoint-client-secret` - the client secret to be used with the token introspection endpoint
+- `post_logout_redirect_uri` - a given URL where the Identity Provider should redirect to after logout
+- `allowed-user-backends` - use this parameter to limit the users which are allowed to login to a specific user backend (e.g., LDAP) if you want to prevent local db users to log in without OpenID Connect => VERIFY + EXAMPLE
+- `use-access-token-payload-for-user-info` - if set to `true`, any user information will be read from the access token. If set to `false`, the userinfo endpoint is used (requires version >= 1.1.0)
+
+=> needs table!
+
+==== Example configuration for Kopano Konnect
+
+Below you find an example snippet that can be added to `config.php`. It is based on the above configuration of the Identity Provider (Kopano Konnect). Other environments might require a different set of parameters and values.
+
+```
+'openid-connect' =>
+array (
+  'provider-url' => 'https://idp.example.com',
+  'client-id' => 'ownCloud',
+  'client-secret' => 'ownCloud',
+  'loginButtonName' => 'Kopano',
+  'autoRedirectOnLoginPage' => false,
+  'redirect-url' => 'https://cloud.example.com/index.php/apps/openidconnect/redirect',
+  'mode' => 'userid',
+  'search-attribute' => 'preferred_username',
+  'use-token-introspection-endpoint' => false,
+),
+```
+
+==== Set up service discovery
 
-A distributed Memcache setup is required to operate this app - such as Redis or Memcached properly.
-Please follow xref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
+In order to allow the ownCloud Clients (Desktop/Android/iOS) to make use of OpenID Connect, ownCloud Server needs to provide service discovery information under the static path `https://cloud.example.com/.well-known/openid-configuration`.
+When enabled, the OpenID Connect App provides the service discovery information on the endpoint `https://cloud.example.com/index.php/apps/openidconnect/config`.
+
+To make the endpoint available under the static service discovery path, it is recommended to put a `RewriteRule` in place using `.htaccess` (the Apache modules `proxy` and `proxy_http` have to be enabled):
```

Comment:
For the sake of completness - this rewrite rule can be setup in the apache config as well ....